airbyte-integrations/connectors/source-clazar/integration_tests/acceptance.py

```diff
@@ -0,0 +1,14 @@
+#
```

Comment:
I.e.  this — I believe you can safely delete the file with no impact to actually running tests.